.evergreen/install-npm-deps.sh

```diff
@@ -8,30 +8,38 @@ set -x
 # Always install top-level dependencies.
 npm ci --verbose
 
+echo "npm packages before required-optional-dependencies"
+npm ls
+
 # In CI, dependencies that are otherwise optional can be required.
 # Mark them as necessary here, so that we get decent error messages
 # if installing one of them fails.
 npm run mark-ci-required-optional-dependencies
 
 if [ x"$NPM_DEPS_MODE" = x"cli_build" ]; then
-  # We always need the build package for various tasks.
-  npm run bootstrap-ci -- --scope @mongosh/build
-
-  # Bootstrap cli-repl + dependencies.
+  # Install cli-repl + dependencies.
   # mongodb-client-encryption cannot be installed everywhere without prerequisites
   # (because it's hard to build from source when there's no prebuilts available
   # because it requires libmongocrypt to be installed globally then), but we still
   # need its types; so if we do not need all packages to be installed,
   # we first try to install mongodb-client-encryption, and if that fails, we fall back
   # to installing with --ignore-scripts (i.e. do not attempt to build addons).
-  (npm run bootstrap-ci -- --scope @mongosh/cli-repl --include-dependencies && test -e packages/service-provider-server/node_modules/mongodb-client-encryption) || \
-    npm run bootstrap-ci -- --scope @mongosh/cli-repl --include-dependencies --ignore-scripts
+  (npm ci -w @mongosh/build -w @mongosh/cli-repl && test -e packages/service-provider-server/node_modules/mongodb-client-encryption) || \
+    npm ci -w @mongosh/cli-repl --ignore-scripts
```

Comment:
What does this do given that we already ran `npm ci` above in line 9?

Reply:
We've been trying to figure out for many hours :)

Reply:
`npm run bootstrap-ci -- --scope @mongosh/cli-repl --include-dependencies` ran `lerna bootstrap --ci --scope @mongosh/cli-repl --include-dependencies`.

lerna bootstrap is apparently just npm install nowadays. --scope is -w. npm install always includes dependencies?